### PR TITLE
More trace logs for TX locking

### PR DESCRIPTION
**Pull request by bfops**
# Description of Changes

Some small changes to make the trace logs more consistent for tx locks.

# API and ABI breaking changes

Nope

# Expected complexity level and risk

1

# Testing

This is a holdover from a debugging session. It was tested-by-fire.

### CODE REVIEW

File: crates/core/src/db/relational_db.rs [hunk 1]
```diff
@@ -440,9 +444,9 @@ impl RelationalDB {
     where
         F: FnOnce(&mut Tx) -> T,
     {
-        let mut tx = self.inner.begin_tx();
+        let mut tx = self.begin_tx();
         let res = f(&mut tx);
-        self.inner.release_tx(ctx, tx);
+        self.release_tx(ctx, tx);
```
Comment (bfops) [inline crates/core/src/db/relational_db.rs:449]: these are so we get the log messages consistently

File: crates/core/src/subscription/module_subscription_manager.rs [hunk 2]
```diff
@@ -125,7 +125,7 @@ impl SubscriptionManager {
         let tables = &event.status.database_update().unwrap().tables;
         let slow = db.read_config().slow_query;
         let tx = scopeguard::guard(db.begin_tx(), |tx| {
-            tx.release(&ExecutionContext::incremental_update(db.address(), slow));
+            db.release_tx(&ExecutionContext::incremental_update(db.address(), slow), tx);
```
Comment (bfops) [inline crates/core/src/subscription/module_subscription_manager.rs:128]: so we get the log messages consistently

File: crates/core/src/db/relational_db.rs [hunk 3]
```diff
@@ -300,7 +304,7 @@ impl RelationalDB {
 
     #[tracing::instrument(skip_all)]
     pub fn release_tx(&self, ctx: &ExecutionContext, tx: Tx) {
-        log::trace!("ROLLBACK TX");
+        log::trace!("RELEASE TX");
```
Comment (bfops) [inline crates/core/src/db/relational_db.rs:307]: I think this was a typo?

File: crates/core/src/db/relational_db.rs [hunk 4]
```diff
@@ -283,13 +283,17 @@ impl RelationalDB {
...
+        log::trace!("ACQUIRED MUT TX");
+        r
     }
 
     #[tracing::instrument(skip_all)]
     pub fn begin_tx(&self) -> Tx {
         log::trace!("BEGIN TX");
-        self.inner.begin_tx()
+        let r = self.inner.begin_tx();
+        log::trace!("ACQUIRED TX");
```
Comment (bfops) [inline crates/core/src/db/relational_db.rs:295]: when we were debugging lock issues, we appreciated not only having a "trying to acquire lock" message but also a "I am now holding the lock" message.

**Review by mamcx — APPROVED** [review]
LGTM

**Pull request merged**